harvard-lts/rabbitmq-email-notifier
# Etd 201: fix pypi packaging, jenkinsfile

** fix pypi packaging, jenkinsfile**
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/ETD-201


# What does this Pull Request do?
fixes the pypi package, updates jenkinsfile to output to dais slack channel

# How should this be tested?

0) copy .env.example to .env, using dev properties
1) start container
2) enter container
3) run `pytest`. all tests should pass

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? y

# Interested parties
Tag (@ mention) interested parties
